langgraph_supervisor/supervisor.py

```diff
@@ -65,6 +80,33 @@ def _process_output(output: dict) -> dict:
             pass
         elif output_mode == "last_message":
             messages = messages[-1:]
+        elif output_mode == "only_history":
+            to_drop = []
+            for m in messages[:]:
+                if (
+                    isinstance(m, ToolMessage)
+                    and m.response_metadata
+                    and (
+                        m.response_metadata.get(METADATA_KEY_HANDOFF_DESTINATION)
+                        or m.response_metadata.get(METADATA_KEY_IS_HANDOFF_BACK)
```

Comment:
technically this is already controlled by the `add_handoff_back_messages` in `create_supervisor` (that can just be set to `False`), but probably fine to just remove here in the same fashion